code/modules/clothing/suits/miscellaneous.dm

```diff
@@ -634,6 +641,57 @@
 /obj/item/clothing/head/hooded/winterhood/security
 	icon_state = "winterhood_security"
 
+/obj/item/clothing/suit/hooded/wintercoat/detective
+	name = "detective winter coat"
+	icon_state = "coatdetective"
+	item_state = "coatdetective"
+	allowed = list(
+		/obj/item/tank/internals,
+		/obj/item/melee/classic_baton,
+		/obj/item/gun/ballistic/revolver/detective,
+		/obj/item/detective_scanner,
+		/obj/item/flashlight,
+		/obj/item/taperecorder
```

Comment:
```suggestion
		/obj/item/taperecorder,
```

Reply:
guh my bad, I'll hold on on fixing this, I thinking I misinterpreted how the lists on subtypes would interact with the original ones.
Are they additive or do they replace completely the parent list?

Reply:
Adding a list to the subtype redefines it. It will not inherit the parents list if you redefine it.

Reply:
Then I'll have to redo it tomorrow ACH, thanks...